appliance_hardening_guide/_topics/db_ssl.md

```diff
@@ -106,6 +93,17 @@ The database appliance now accepts only connections from connecting appliances t
         vmdb_production=#
 
     The `psql` displays information about the SSL connection, which indicates that the configuration succeeded. Enter `\q` to leave `psql`.
+    The `PGSSLMODE` variable forces a certain level of SSL. To learn more, please see [postgress sslmode documentation].
+
+    [postgress sslmode documentation]: https://www.postgresql.org/docs/14/libpq-connect.html#LIBPQ-CONNECT-SSLMODE
+
+5.  Ensure `/etc/default/manageiq.properties` has the proper ssl key file location defined:
+
+        $ `grep PGSSLROOTCERT /etc/default/manageiq.properties ` || \
```

Comment:
Super minor, but these backticks within a code block are just weird to me.  Minor, but can we do:

```suggestion
        $ $(grep PGSSLROOTCERT /etc/default/manageiq.properties) || \
```